src/database/schema/ChatSchema.ts

```diff
@@ -224,7 +237,7 @@ class ChatSchema implements BaseDbSchema {
       channelId,
       userId: json?.user?.id,
       message: (json?.text || json?.message) ?? '',
-      type: json?.type,
+      type: json?.message_type ?? json?.type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to line 214, this change adds a fallback for the `type` field to use `json?.message_type` if `json?.type` is undefined. This looks fine as long as the `message_type` and `type` fields are intended to serve the same purpose.


<!-- This is an auto-generated comment by OSS CodeRabbit -->